synapse/swarm/opers/basic.py

```diff
@@ -251,6 +257,31 @@ def _oper_lift(self):
             for tufo in self.query.getTufosByPropFrom(newprop,valu=valu,fromtag=ftag):
                 self.query.add(tufo)
 
+    def run(self):
```

Comment:
Somewhere, my subconscious is yelling at me that "if a simple Oper like pivot requires a special case, we've designed/implemented metrics wrong".  I think by tracking add/take/clear etc ( and recording number of adds, number of dels, and final total ) we probably dont end up needing a special case for pivot().